playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -307,7 +307,7 @@ private void sendDistinctEvent(PlayerEvent.Type newEvent) {
     }
 
     private void sendEvent(PlayerEvent.Type event) {
-        if (shouldRestorePlayerToPreviousState) {
+        if (shouldRestorePlayerToPreviousState && event != PlayerEvent.Type.PLAY) {
```

Comment:
I think that for now it is safer to do what we did with Youbora with the `onApplicationResumed` and check if player is playing there and emit the play media mark